pkgmgmtutils/formula_updater_test.go

```diff
@@ -115,8 +115,8 @@ var _ = Describe("FormulaUpdater", func() {
 						PRDescription:   "Created by Solo-io Bot",
 						PRCommitName:    "Solo-io Bot",
 						PRCommitEmail:   "bot@solo.io",
-						VersionRegex:    `:tag\s*=>\s*"v([0-9.]+)",`,
-						VersionShaRegex: `:revision\s*=>\s*"(.*)"`,
+						VersionRegex:    `tag\s*=>\s*"v([0-9.]+)",`,
```

Comment:
is there a reason this regex is different from the regex here: https://github.com/solo-io/go-utils/pull/435/files#diff-0f08ec98106c6b85f243a4383477e6545c44dde83f05e345e80240e5484a397dR266

Reply:
No they should be the same. The formula_updater_test doesn't called UpdateFormulaBytes so the regex here doesn't matter for the test, but it should still match.